src/sqlfluff/cli/commands.py

```diff
@@ -483,13 +484,14 @@ def fix(force, paths, processes, bench=False, fixed_suffix="", logger=None, **kw
         if result.num_violations(types=SQLLintError, fixable=True) > 0:
             stdout = result.paths[0].files[0].fix_string()[0]
         else:
-            if verbose:
-                if templater_error:
-                    click.echo("Fix aborted due to unparseable template variables.")
-                if unfixable_error:
-                    click.echo("Unfixable violations detected.")
             stdout = stdin
 
+        if verbose:
+            if templater_error:
+                click.echo("Fix aborted due to unparseable template variables.")
+            if unfixable_error:
+                click.echo("Unfixable violations detected.")
```

Comment:
I thought better to raise this error even if there are fixable errors.

Reply:
AH yeah good call! That was the real spirit of the issue anyway.